### ISSUE TITLE
webpack-bundle-analyzer errors out on differential build - no such file

### ISSUE THREAD

**Issue by SanderElias** [angular/angular-cli#15915]
# 🐞 Bug report
As I wanted to analyze my bundles I added `webpack-bundle-analyzer` to my project.
Then I used the command
```bash
ng build --prod --statsJson && webpack-bundle-analyzer ./dist/app/stats-es2015.json
```
result:
![image](https://user-images.githubusercontent.com/1249083/67312428-9f6cf080-f501-11e9-9cfb-71490b2a6168.png)

This happened in mutliple of my projects. The common demoninator seems to be diffential loading. As soon as that is turned on, I get this error.

**Comment by SanderElias** [comment]
For anyone that needs a workaround:
```bash
ng build --prod --source-map && npx source-map-explorer ./dist/app/*-es2015.*.js
```
uses a different tool that also analyzes your bundles.

**Comment by dgp1130** [comment]
From Charles: Seems that this one file gets renamed after the Webpack stats are generated. Future rearchitecting should fix problems like this in the longer term. For v9, we can just rename this one file, which should be "good enough" for the time being.

https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts#L287

**Comment by dgp1130** [comment]
Sent out https://github.com/angular/angular-cli/pull/16169 with a hack to fix this in the short term.

**Comment by dgp1130** [comment]
I closed PR #16169 in favor of a better idea @clydin had in #16181. Passing this issue to him.
